.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,35 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ next, beta, master ]
+  pull_request:
+    branches: [ next, beta, master ]
+  schedule:
+    - cron: '39 21 * * 6'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
```

Comment:
I think this part is unnecessary as there is only one job in this file and we can explicitly declare the language below